cli/src/semgrep/run_scan.py

```diff
@@ -635,13 +635,15 @@ def run_scan(
                     baseline_target_strings = target_strings
                     baseline_target_mode_config = target_mode_config
```

Comment:
We need the Pro PR to not be a draft one and be all green (except for the "check-submodule" workflow.

Reply:
done.